docs/source/contributing/prerequisites.md

```diff
@@ -1,4 +1,8 @@
-# Prerequisites
+---
+title: "Prerequisites"
+sidebar_title: "System Prerequisites"
+description: "Before contributing..."
```

Comment:
I think we could improve this description text. Maybe "setting up your development environment"